kafka/producer/base.py

```diff
@@ -139,7 +143,8 @@ def __init__(self, client, async=False,
             log.warning("async producer does not guarantee message delivery!")
             log.warning("Current implementation does not retry Failed messages")
             log.warning("Use at your own risk! (or help improve with a PR!)")
-            self.queue = Queue()  # Messages are sent through this queue
+            # Messages are sent through this queue
+            self.queue = Queue(maxsize=batch_send_queue_maxsize)
```

Comment:
this is used both in batch mode as well as in simple async mode -- the variable name probably should not be specific to batch mode